### ISSUE TITLE
Add option to connect to a website with a self-sign cert

### ISSUE THREAD

**Issue by nifker** [beakerbrowser/beaker#860]
Operation System: All
Beaker Version: 0.7.11
Let the user decide whether they want to connect to a website with a

**Comment by pfrazee** [comment]
Yeah this is an existing issue. https://github.com/beakerbrowser/beaker/issues/419. I'm going to keep this open even though it's a duplicate, so it stays visible. Thanks for filing.
